features/admin.claims.v1/components/edit/external-dialect/edit-external-claims.tsx

```diff
@@ -137,7 +137,12 @@ export const EditExternalClaims: FunctionComponent<EditExternalClaimsPropsInterf
     ];
 
     const featureConfig: FeatureConfigInterface = useSelector((state: AppState) => state.config.ui.features);
-    const allowedScopes: string = useSelector((state: AppState) => state?.auth?.allowedScopes);
+    const attributeDialectFeatureConfig: FeatureAccessConfigInterface = useSelector((state: AppState) =>
+        state.config.ui.features.attributeDialects);
+    const oidcScopesFeatureConfig: FeatureAccessConfigInterface = useSelector((state: AppState) =>
+        state.config.ui.features.oidcScopes);
+    const hasDialectCreateScopes: boolean = useRequiredScopes(attributeDialectFeatureConfig?.scopes?.create);
```

Comment:
Let's follow the naming convention mentioned in the docs
https://github.com/wso2/identity-apps/blob/bc340d7c0781dc6ee13eae177612baa8748e09d0/docs/write-code/CODE_QUALITY.md?plain=1#L67

ex: `hasDialectCreatePermissions`

Reply:
Fixed with e3df748c1b7614f75fda24bb84a30038750488c2